third_party_integration/huggingface_transformers/0001-Modifications-for-NNCF-usage.patch

```diff
@@ -1189,7 +1189,7 @@ index 5b000c81..05df28a7 100644
  
 +        if nncf_config is not None and nncf_eval:
 +            compression_algo_controller, model = create_compressed_model(model, nncf_config,
-+                                                                         resuming_state_dict=state_dict)
++                                                                         compression_state=state_dict)
```

Comment:
simply load olds-style checkpoint for now,
will save new-style checkpoints as a separate feature. task 56741